R/drive_user.R

```diff
@@ -38,10 +38,10 @@ guser <- function(fields = "user", ...) {
     return(NULL)
   }
 
-  request <- build_request(
-    endpoint = "drive.about.get",
-    params = list(fields = fields,
-                  ...))
+  request <-  list(url = httr::modify_url(.drive$base_url, path = paste0("drive/v3/about")),
+                   query = list(fields = fields),
+                   method = "GET",
+                   token = drive_token())
   response <- make_request(request)
```

Comment:
Shouldn't this also be one of our endpoints? Seem like we could do same for "About" as we're doing for files, permission, and revisions.